packages/integrations/cloudflare/src/index.ts

```diff
@@ -150,7 +150,11 @@ export default function createIntegration(args?: Options): AstroIntegration {
 						.map((file: string) => `/${file}`);
 
 					for (let page of pages) {
-						staticPathList.push(prependForwardSlash(page.pathname));
+						let pagePath = prependForwardSlash(page.pathname);
+						if (config.base !== '/') {
+							pagePath = `${config.base}${pagePath}`;
```

Comment:
This is making an assumption that `base` doesn't have a trailing slash. Should I account for that?